db/models.py

```diff
@@ -14,3 +16,37 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
```

Comment:
The `related_name` for the `actors` field in the `Movie` model is set to 'actors', which is the same as the field name. This can lead to confusion and potential conflicts. Consider using a different `related_name`, such as 'movies' or 'movie_actors', to better reflect the reverse relationship.